aclk/README.md

```diff
@@ -1,25 +1,103 @@
+the features found in Netdata Cloud."
+date: 2020-03-26
 custom_edit_url: https://github.com/netdata/netdata/edit/master/aclk/README.md
 ---
 -->
 
 # Agent-cloud link (ACLK)
 
+The agent-cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud. The ACLK uses
+[MQTT](https://en.wikipedia.org/wiki/MQTT) over secure websockets to create, persist, and encrypt the connection from
+end-to-end, and enable the features found in Netdata Cloud.
 
-## Configuration Options
+The ACLK is a vital component of the entirely redesigned and re-architected Netdata Cloud.
 
-In `netdata.conf`:
+_No data is exchanged with Netdata Cloud until you claim a node._ By claiming a node, you opt-in to sending data from
+your agent to Netdata Cloud via the ACLK. Data used to render charts and alarms in Netdata Cloud is accessed directly
+from agents through the ACLK. Though the data does flow through Netdata servers on its way from agents to the browser,
+it is never logged or stored.
 
-```ini
+Read our [claiming documentation](../claim/README.md) for details on node claiming.
+
+## Enable and configure the ACLK
+
+The ACLK is enabled by default if its prerequisites installed correctly, and the correct configuration is already set in
+your agent's `netdata.conf` file.
+
+```conf
+[cloud]
+    cloud base url = https://app.netdata.cloud
+```
+
+Unless your node uses a proxy to connect with the internet, you do not need to configure the ACLK further. See the next
+step for proxy configuration.
+
+### Proxy configuration
+
+If your Netdata agent uses a proxy to reach the outside internet, you must configure a SOCKS5 or HTTP proxy in the
+`[cloud]` section of your `netdata.conf` file. A SOCKS5 proxy is preferred. By default, the section looks
+like the following:
+
+```conf
 [cloud]
     proxy = none
 ```
 
-Parameter proxy can take one of the following values:
+The `proxy` setting takes a few different values:
+
+-   `none`: Do not use any proxy, even if the system is configured otherwise.
+-   `env`: Netdata reads the environment variables `socks_proxy` and `http_proxy` to discover the correct
+    proxy settings.
+-   `socks5[h]://[user:pass@]host:ip`: Netdata uses the specified SOCKS proxy.
+-   `http://[user:pass@]host:ip`: Netdata uses the specified HTTP proxy.
+
+## Disable the ACLK
+
+If you would prefer to disable the ACLK and not use Netdata Cloud, you have two options:
+
+1.  Pass `--disable-cloud` to `netdata-installer.sh` during installation. When you pass this parameter, the installer
+    does not download or compile any extra libraries, and the agent behaves as though the ACLK, and thus Netdata Cloud,
+    does not exist.
+
+2.  Change a runtime setting in your `netdata.conf` file. This setting only stops the agent from attempting any
+    connection via the ACLK, but does not prevent the installer from downloading and compiling the ACLK's dependencies.
```

Comment:
Maybe would add a sentence at the end?:
"This means ACLK functionality is available in the agent but remains fully inactive."

Reply:
Additionally by default this key is set to `coming soon` in agent. This means now  same as `disable` and after the cloud release it will mean same as `enable`. Should we inform user about that? (to not make them confused when they see it in config?)

Reply:
Hmmm, OK I guess then people who have config with `coming soon` in it from previous versions will be a minority or they will just rewrite it to `enable`/`disable` upon noticing it.